spec/integration/recipe_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Public Recipe show', type: :system do
+  describe 'index page' do
+    before(:example) do
+      @user = User.create(
+        name: 'Gabriel',
+        email: 'didier@gmail.com',
+        password: 'anything',
+        confirmed_at: Time.now.utc
+      )
+
+      # sign_in @user
```

Comment:
[OPTIONAL] Kindly remove this line if it's not used code. Thank you!

Reply:
Thanks @Rustamxon7 for the suggestion.